packages/xo-web/src/common/intl/messages.js

```diff
@@ -908,6 +908,13 @@ const messages = {
   poolSupportSourceUsers: 'XCP-ng Pro Support not available for source users',
   poolSupportXcpngOnly: 'Only available for pool of XCP-ng hosts',
   poolLicenseAlreadyFullySupported: 'The pool is already fully supported',
+  rollingPoolReboot: 'Rolling pool reboot',
```

Comment:
We usually capitalize feature names:
```suggestion
  rollingPoolReboot: 'Rolling Pool Reboot',
```